packages/components/addon/components/hds/dropdown/index.js

```diff
@@ -6,14 +6,21 @@
 import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
 
-export const DEFAULT_POSITION = 'right';
-export const POSITIONS = ['right', 'left'];
+export const DEFAULT_POSITION = 'bottom-right';
+export const POSITIONS = [
+  'right',
```

Comment:
maybe add a comment (and a link to this PR) to explain why we still show these old values